.github/workflows/wheels.yml

```diff
@@ -86,6 +86,12 @@ jobs:
             python: 312
             platform_id: manylinux_x86_64
             manylinux_image: manylinux2014
+          - os: ubuntu-latest
+            python: 313t
+            platform_id: manylinux_x86_64
+            manylinux_image: manylinux2014
+            prerelease_pythons: True
```

Comment:
Please add TODO to make it explicit that this should go away once CPython 3.13 is out.